### PR TITLE
Debian/master

### PR DESCRIPTION
**Pull request by sdelafond**
Description of the issue/feature this PR addresses: Debian packaging for inclusion in sid

Desired behavior after PR is merged: make sure the next odoo release is based on what will be in Debian sid

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by d-fence — CHANGES_REQUESTED** [review]
Thanks for this PR.
May I ask you to change your commit message according to [our guidelines](https://www.odoo.com/documentation/14.0/reference/guidelines.html#commit-message-structure).
In this particular case, the first line of the commit messages should be something like:
`[FIX] packaging: fix debian files for 14.0.0 release`
and 
`[FIX] packaging: update debian changelog`
and
`[CLA] signature for sdelafond`

Also, the qweb ast validation patch is not needed anymore.

File: debian/patches/0001-fix-qweb-ast-validation.patch [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+         ]
+@@ -1550,7 +1550,7 @@ def _compile_directive_call(self, el, options):
+                     if isinstance(key, str):
+                         keys.append(ast.Str(s=key))
+                     elif key is None:
+-                        keys.append(ast.Name(id='None', ctx=ast.Load()))
++                        keys.append(ast.Constant(None))
+                     values.append(ast.Str(s=value))
+ 
+                 # {'nsmap': {None: 'xmlns def'}}
```
[the remark this answers is not in the document]
Comment (sdelafond) [reply debian/patches/0001-fix-qweb-ast-validation.patch]: Done.

**Comment by sdelafond** [comment]
Should be OK now.

**Comment by d-fence** [comment]
> Should be OK now.

Hmm, one more thing, can you squash cf8d6f6 with da98f49 to avoid useless commits ?
Thanks

**Comment by sdelafond** [comment]
> > Should be OK now.
> 
> Hmm, one more thing, can you squash [cf8d6f6](https://github.com/odoo/odoo/commit/cf8d6f698b300b00acf71cfae206617c08f82ef1) with [da98f49](https://github.com/odoo/odoo/commit/da98f4989799c22e5fd4bb2bbd92be0163c6abb8) to avoid useless commits ?

If possible, I'd rather not, as da98f4 is what was uploaded to NEW, and as such shouldn't disappear from the history (it holds the debian/14.0.0 tag).

**Comment by d-fence** [comment]
Will be merged with #59946

**Pull request closed without merging**